tests/ui/error-codes/E0452.rs

```diff
@@ -2,5 +2,7 @@
                     //~| ERROR E0452
                     //~| ERROR E0452
                     //~| ERROR E0452
+                    //~| ERROR E0452
```

Comment:
Duplicated attr errors is due to a call to `lints_that_dont_need_to_run` in check-builds. This is deduplicated to the user, so it doesn't matter.